cypress/integration/articleBodySpec.js

```diff
@@ -0,0 +1,129 @@
+import config from '../support/config/services';
+import {
+  copyrightDataWindow,
+  firstHeadlineDataWindow,
+  firstParagraphDataWindow,
+  firstSubheadlineDataWindow,
+  getElement,
+  placeholderImageLoaded,
+  renderedTitle,
+  visibleImageNoCaption,
+  visibleImageWithCaption,
+  shouldContainText,
+} from '../support/bodyTestHelper';
+
+const serviceHasArticlePageType = service =>
```

Comment:
You only use this once, there's no need to factor it out into a const.

Reply:
I suggested this for readability

Reply:
See comment above

Reply:
Fixed the package lock changes.